en/docs/learn/try-authorization-code-grant.md

```diff
@@ -66,54 +65,21 @@ without PKCE.
             ``` 
 
 3.  Log in with the user credentials.  
-    ![log-into-app](../assets/img/using-wso2-identity-server/log-into-app.png) 
-
-      
+    ![log-into-app](../assets/img/learn/login-oauth.png) 
 
-4.  Select **Approve Once** or **Approve Always** in Access to profile
-    information section. Also, select the attributes you agree to share.
-    Click **Continue**.  
-    ![select-attributes-and-consent](../assets/img/using-wso2-identity-server/select-attributes-and-consent.png) 
+4.  Click **Allow**.
+    ![select-attributes-and-consent](../assets/img/learn/remember-consent.png) 
 
-      
+5.  Provide the following details and click **Get Access Token**.
 
-5.  Provide the following details and click on **Get Access Token**.
+    -   Callback URL: `http://wso2is.local:8080/playground2/oauth2client`
```

Comment:
the callback url should reflect `localhost.com` or `localhost` (depending on which is correct) instead of `wso2is.local`